README.md

```diff
@@ -1,24 +1,24 @@
 You will receive a compass that will point you towards the runner who is closest to you. The compass will only update when you hold it in your main hand.  
-In case, no runner is in your current dimension, it will still point to the last known location of any runner that was closest to you last. Runner locations are saved for you per dimension, so even if you switch a dimension, the compass will still point you to the runner you tracked last in that dimension.
-All of this works with multiple hunters. The compass will track the closest runner per hunter.
+In case no Runner is in your current dimension, it will still point to the last known location of the runner that was closest to you. Runner locations are saved for you per dimension, so even if you switch a dimension, the compass will still point you to the runner you tracked last in that dimension.
+All of this works with multiple hunters. The compass will track the closest runner for each Hunter.
 
 ## Runners
-As a runner, your goal is to kill the Ender Dragon.  
-It doesn't matter who kills it, so even if it receives its killing blow from a hunter, it counts as a win towards the Runners.  
-You only have one live. If you die, may it be a pvp kill by a hunter, by pve, or through enviornmental damage, you are out. If you play this with multiple runners, you as a team can still win.  
-You can still regenerate health like usual with a full hunger bar, so this is **not** UHC for you.
+As a Runner your goal is to kill the Ender Dragon.  
+It doesn't matter who kills it, even if it receives it's killing blow from a Hunter, it counts as a win towards The Runners.  
+You only have one life! If you die, may it be a pvp kill by a hunter, from pve or through enviornmental damage, you are out. If you play this with multiple Runners, you as a team can still win.  
+You can regenerate health like usual with a full hunger bar, so this is **not** UHC for you.
 
 ## Rules
 1. There are no rules  
-Do anything you can think of to gain an advantage. This may include trapping the obsidian platform in the end with lava. Just have a water bucket selected in your toolbar and be ready to run, I guess :wink:. The game will fill a certain area above the obsidian platform with air whenerver a player goes through, so you will have a sort of grace periode.  
+Do anything you can think of to gain an advantage. This may include trapping the obsidian platform in the end with lava. Just have a water bucket selected in your toolbar and be ready to run, I guess :wink:. The game will fill a certain area above the obsidian platform with air whenerver a player goes through, so you will have a sort of grace period.  
 Of course, you as a group can agree on any rules you seem fit.
 
-It is recommended to have more experienced players join the runners and less so the hunters.
+It is recommended to have the more experienced players join the Runners and The less so The Hunters.
```

Comment:
> The less so
In that case `the` should be lowercase.

Reply:
Or with experienced players it should be "less experienced" without "so" :D

Reply:
Whoops changed the wrong "the". Will fix as soon as I figure out gh's UI again